x/withdraw/keeper/utils_test.go

```diff
@@ -0,0 +1,53 @@
+package keeper_test
+
+import (
+	"github.com/stretchr/testify/mock"
+
+	tmbytes "github.com/tendermint/tendermint/libs/bytes"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	bankkeeper "github.com/cosmos/cosmos-sdk/x/bank/keeper"
+
+	transfertypes "github.com/cosmos/ibc-go/v3/modules/apps/transfer/types"
+	clienttypes "github.com/cosmos/ibc-go/v3/modules/core/02-client/types"
+
+	"github.com/tharsis/evmos/v2/x/withdraw/types"
+)
+
+/*
+  Test objects
+*/
+
+var _ types.TransferKeeper = &TransferKeeper{}
+
+// TransferKeeper is a mocked object that implements an interface
+// that describes an object that the code I am testing relies on.
+type TransferKeeper struct {
```

Comment:
```suggestion
type MockTransferKeeper struct {
```

It is best to be explicit about what is mocked, versus what isn't.